packages/config/src/layer2s/zksyncera.ts

```diff
@@ -78,8 +68,9 @@ export const zksyncera: Layer2 = {
     upgradeability: RISK_VIEW.UPGRADABLE_YES,
     sequencerFailure: RISK_VIEW.SEQUENCER_NO_MECHANISM,
     validatorFailure: {
-      ...RISK_VIEW.PROVER_DOWN,
-      description: 'Only whitelisted Validators can submit proofs.',
+      value: 'In development',
+      description: 'Currently, there is no generic escape hatch.',
+      sentiment: 'bad',
```

Comment:
1. Get the list of permissioned Validators from the discovery

2. Describe in full details what happens when Validator stops submitting proofs